docs/source/notebooks_and_ipython/kedro_and_notebooks.md

```diff
@@ -32,35 +34,41 @@ We recommend that you save your notebook in the `notebooks` folder of your Kedro
-* `context` (type `KedroContext`): Kedro project context that provides access to Kedro's library components
+* `context` (type `KedroContext`): [Kedro project context](../api/kedro.framework.context.rst) that provides access to Kedro's library components
 * `pipelines` (type `Dict[str, Pipeline]`): Pipelines defined in your [pipeline registry](../nodes_and_pipelines/run_a_pipeline.md#run-a-pipeline-by-name)
 * `session` (type `KedroSession`): [Kedro session](../kedro_project_setup/session.md) that orchestrates a pipeline run
 
 ``` {note}
 If the Kedro variables are not available within your Jupyter notebook, you could have a malformed configuration file or missing dependencies. The full error message is shown on the terminal used to launch `kedro jupyter notebook`.
 ```
 
-## How to explore a Kedro project in a notebook
-Here are some examples of how to work with the Kedro variables. To explore the full range of attributes and methods available, see the relevant [API documentation](/api/kedro) or use the [Python `dir` function](https://docs.python.org/3/library/functions.html#dir), for example `dir(catalog)`.
-
-### `%run_viz` line magic
+## Loading the project with the `kedro.ipython` extension
 
-``` {note}
-If you have not yet installed [Kedro-Viz](https://github.com/kedro-org/kedro-viz) for the project, run `pip install kedro-viz` in your terminal from within the project directory.
+A quick way to explore the `catalog`, `context`, `pipelines`, and `session` variables in your project within a IPython compatible environment, such as Databricks notebooks, Google Colab, and more, is to use the `kedro.ipython` extension.
+This is tool-independent and useful in situations where launching a Jupyter interactive environment is not possible. You can use the [`%load_ext` line magic](https://ipython.readthedocs.io/en/stable/config/extensions/index.html#using-extensions) to explicitly load the Kedro IPython extension:
+```ipython
+In [1]: %load_ext kedro.ipython
 ```
 
-You can display an interactive visualisation of your pipeline directly in your notebook using the `run-viz` [line magic](https://ipython.readthedocs.io/en/stable/interactive/magics.html) from within a cell:
+If you have launched your interactive environment from outside your Kedro project, you will need to run a second line magic to set the project path.
+This is so that Kedro can load the `catalog`, `context`, `pipelines` and `session` variables:
 
-```python
-%run_viz
+```ipython
+In [2]: %reload_kedro <project_root>
 ```
+The Kedro IPython extension remembers the project path so that future calls to `%reload_kedro` do not need to specify it:
 
-![View your project's Kedro Viz inside a notebook](../meta/images/run_viz_in_notebook.png)
+```ipython
+In [1]: %load_ext kedro.ipython
+In [2]: %reload_kedro <project_root>
+In [3]: %reload_kedro
```

Comment:
Why we need to reload so many times?

Reply:
I think this is to show that 2 and 3 are the same? Given the content of the docs above it